python/az/aro/azext_aro/_params.py

```diff
@@ -64,10 +63,6 @@ def load_arguments(self, _):
         c.argument('service_cidr',
                    help='CIDR of service network. Must be a minimum of /18 or larger.',
                    validator=validate_cidr('service_cidr'))
-        c.argument('software_defined_network', arg_type=get_enum_type(['OVNKubernetes', 'OpenShiftSDN']),
```

Comment:
is there a deprecation process for this type of change? or are we free to rip it out as we want?

Reply:
it's not in the API that the aro command module uses anymore so I think we're just good to rip it out.  It's not even honored if you pass it anyway right now so we should remove it to remove confusion as well.